images/algod/DevModeNetwork.json

```diff
@@ -1,6 +1,5 @@
 {
     "Genesis": {
-        "ConsensusProtocol": "future",
```

Comment:
Why this change? 

Maybe misreading but would cause the proto version to be the latest from here:  https://github.com/algorand/go-algorand/blob/master/gen/generate.go#L77

Reply:
Future enables unreleased features in algod which increases the chance that Indexer would be incompatible.

For example, I was recently running the current Indexer against an old version of algod which has the"compact certificate" feature. It doesn't work without this change.

Reply:
Fair enough,  it might confuse some exciting folks at first but def better for harder to debug issues. thanks!